js/page-specific/homepage/featured-events.js

```diff
@@ -0,0 +1,98 @@
+namespacer('bcpl.pageSpecific.homepage');
+
+bcpl.pageSpecific.homepage.featuredEvents = (($, Handlebars, moment, CONSTANTS) => {
+	const activatePost = (event) => {
+		const $target = $(event.currentTarget);
+		const $animationTarget = $target.find('.animated');
+
+		$animationTarget.addClass('active');
+	};
+
+	const deactivatePost = (event) => {
+		const $target = $(event.currentTarget);
+		const $animationTarget = $target.find('.animated');
+
+		$animationTarget.removeClass('active');
+	};
+
+	const formatTime = unformattedTime => {
+		return unformattedTime.replace(':00', '').replace(/\w\w$/, foundString => foundString.split('').join('.') + '.');
```

Comment:
I would make newlines for this, but thats just my preference I don't think it's that important.